### PR TITLE
Ports new Bay SolGov suits as Commonwealth suits

### PR DESCRIPTION
**Pull request by Arokha**
Sprites originally by Pawn (SUPERBOREDGUY) and AzlanOnPC/SomeAngryMiner), derivatives by me for adding the tails 'n such for our species.

Lots of suits have their 'alt' versions, so now they have 'alt2' versions, which are typically even better than the 'alt' versions. Did you know the 'alt' versions are almost always better? Usually better armor and less slowdown. This continues that further, usually with less slowdown.

Z6YSC5oCwD
From left to right:
- commonwealth atmos voidsuit
- commonwealth eng voidsuit
- commonwealth captain voidsuit
- commonwealth mining voidsuit
- commonwealth anomaly suit
- commonwealth pilot voidsuit
- commonwealth engineering hardsuit
- commonwealth medical hardsuit
- commonwealth null hardsuit
- commonwealth pathfinder hardsuit

CUpqrsianr
- commonwealth explorer voidsuit
- commonwealth medical voidsuit
- uhhh those other 3 are the first 3 from the first picture again oops

The voidsuits are what you'd expect. They're voidsuits. I had to make all the ANIMAL PEOPLE sprites. I did my best.
The hardsuits are:
- Engineering: Good modules, competes with industrial hardsuit
- Medical: Faster than normal medical hardsuit, same modules
- Null: Basically a voidsuit, has a full body piece and head piece like a voidsuit, only module is jetpack. Comes loaded from cargo and is much less expensive than all the other hardsuits.
- Pathfinder: Event/reward territory. Huge tank with eblade.

If you like the sprites you can just refit your suit to them. If you want the real ones with the real stats, you can order them from cargo.

### Other things I did in this PR
- Suit cyclers now use a big ol datum list thing to decide what suits they can serve, not a bunch of messy text. Vastly reduced the code bloat in suit cyclers. They should be much easier to add suits to now. Just copypaste one of the datums for existing suits.
- Improved the little 'stand and spin' dummies a little for demo purposes.
- Made a few lists lazy.
- Moved all vorestation species to _vr icon files. For example akula were mob_akula.dmi but now they're mob_vr_akula.dmi, there's no 'non-vr' akula.
- Fixed a bug with rigs that could happen if you disconnected while deploying it (or if your rig was equipped by a fake person like a dummy).

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig/suits/station_vr.dm [hunk 1]
```diff
@@ -153,28 +153,259 @@
...
+	default_mob_icon = 'icons/mob/rig_back_vr.dmi' // the onmob
+	icon_state = "engineering_rig_bay"
+	item_state = null
+	suit_type = "engineering hardsuit"
+	cell_type = /obj/item/weapon/cell/super
+
+	chest_type = /obj/item/clothing/suit/space/rig/bayeng
+	helm_type = /obj/item/clothing/head/helmet/space/rig/bayeng
+	boot_type = /obj/item/clothing/shoes/magboots/rig/bayeng
+	glove_type = /obj/item/clothing/gloves/gauntlets/rig/bayeng
```
Comment (Hatterhat) [inline code/modules/clothing/spacesuits/rig/suits/station_vr.dm:304]: give this mf insuls

**Comment by Detective-Google** [comment]
pog

File: code/modules/clothing/spacesuits/rig/suits/station_vr.dm (hunk 1 above)
Comment (Hatterhat) [reply code/modules/clothing/spacesuits/rig/suits/station_vr.dm:304]: isnt the industrial hardsuit the one miners ~don't use~ can buy from the vendor anyway? if its for engineering purposes compare with the EVA hardsuit

Comment (Arokha) [reply code/modules/clothing/spacesuits/rig/suits/station_vr.dm:304]: The one in the vendor is a special, nerfed version, the `/vendor` subtype.
I mostly went based on the 'real' industrial one. Though I could have gone with the CE's hardsuit I guess...

Comment (Hatterhat) [reply code/modules/clothing/spacesuits/rig/suits/station_vr.dm:304]: huh. so it is.

anyways if it's intended to be used by engineering it should still probably get insuls anyway

Comment (Arokha) [reply code/modules/clothing/spacesuits/rig/suits/station_vr.dm:304]: Ye fair, I'm away from computer but I'll do that tomorrow.

**Pull request merged**